configs/permissions/drift.json

```
@@ -1,3 +1,5 @@
 {
     "*": ["*"]
+    "comparisons": ["read"]
```

Looks like you're missing a comma here. 🙂 
```suggestion
    "comparisons": ["read"],
```

---

updated here and the line above, thanks